README.md

```diff
@@ -21,10 +21,40 @@ To build the image from the Dockerfile, run
 sudo docker build -t analythium/shinyproxy-demo .
 ```
 
-Test locally
+
+
+## Develop and test locally
+```bash
+docker-compose up --build
```

Comment:
`docker-compose up --build` as opposed to the previous `docker run -p 4000:3838 analythium/shinyproxy-demo`.

one of the main differences is that now when you make changes to your app, reloading the page is enough to immediately see the changes.